CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.14.2
```

Comment:
you will want to bump to 6.15.0 since this is an additive change and not a bug fix.